src/common/test/object_table_tests.c

```diff
@@ -146,7 +146,7 @@ int lookup_failed = 0;
 
 void lookup_done_callback(object_id object_id,
                           int manager_count,
-                          OWNER const char *manager_vector[],
```

Comment:
It seems that in one of the recent commits, `manager_vector` now gets freed by the hiredis callback, not the done/fail callbacks.